CONTRIBUTING.md

```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
 Instead, start by [opening an issue](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues) describing the problem you want to solve and how you plan to approach the problem. 
 This will let us have a brief discussion about the problem and, hopefully, identify some potential pitfalls before too much time is spent.
 
-Note:  If you wish to work on something that already exists on our backlog, you can use that work item as your proposal.  
+Note:  If you wish to work on something that already exists on our backlog, you can use that work item as your proposal.
+Please take ownership of the issue so that others do not duplicate your effort.
```

Comment:
I believe only those with write access can [assign issues](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/).

Reply:
Fair enough. I will pull that line.